packages/lib/src/footer/Footer.tsx

```diff
@@ -148,43 +69,43 @@ const Copyright = styled.div`
 `;
 
 const LogoContainer = styled.span<{ mode?: FooterPropsType["mode"] }>`
-  max-height: ${(props) => (props?.mode === "default" ? props.theme.logoHeight : "16px")};
-  width: ${(props) => props.theme.logoWidth};
+  max-height: ${(props) => (props?.mode === "default" ? "var(--height-m)" : "var(--height-xxs)")};
+  width: auto;
 `;
 
 const LogoImg = styled.img<{ mode?: FooterPropsType["mode"] }>`
-  max-height: ${(props) => (props?.mode === "default" ? props.theme.logoHeight : "16px")};
-  width: ${(props) => props.theme.logoWidth};
+  max-height: ${(props) => (props?.mode === "default" ? "var(--height-m)" : "var(--height-xxs)")};
+  width: auto;
 `;
 
 const SocialAnchor = styled.a<{ index: number }>`
-  border-radius: 4px;
+  border-radius: var(--border-radius-s);
 
   &:focus {
-    outline: 2px solid #0095ff;
-    outline-offset: 2px;
+    outline: var(--border-width-m) var(--border-style-default) var(--border-color-secondary-medium);
+    outline-offset: var(--border-width-m);
   }
 `;
 
 const SocialIconContainer = styled.div`
   display: flex;
   align-items: center;
-  color: ${(props) => props.theme.socialLinksColor};
+  color: var(--color-fg-neutral-bright);
   overflow: hidden;
-  font-size: ${(props) => props.theme.socialLinksSize};
+  font-size: var(--height-s);
 
   svg {
-    height: ${(props) => props.theme.socialLinksSize};
-    width: ${(props) => props.theme.socialLinksSize};
+    height: var(--height-s);
+    width: 24px;
   }
 `;
 
 const BottomLinks = styled.div`
   display: inline-flex;
   flex-wrap: wrap;
   align-self: center;
-  padding-top: ${(props) => props.theme.bottomLinksDividerSpacing};
-  color: #fff;
+  padding-top: var(--spacing-padding-xs);
```

Comment:
The same as for the copyright. This padding should be in the parent container or a margin if it remains here.